.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,4 +23,15 @@ test/README.md and add these additional items:
 - [ ] QE has approved reproducer/new tests and
 - [ ] Subsequent commits provide bug fixes without modifying the reproducer; CI reports PASS and
 - [ ] QE approves this PR; RHBZ status is set to `MODIFIED + Verified=Tested`
+
+Information regarding our CI:
```

Comment:
CI sounds confusing because github actions are technically a part of our CI. Let's maybe use the term "GitLab pipeline" in this description? One mention of `Schutzbot` would be also nice to clear any confusion.